### PR TITLE
enable eager option in estimator.fit and test

### PR DESCRIPTION
**Pull request by vbvg2008**
Also make the unit test print warning instead of failing if it's due to data url issue.

### CODE REVIEW

File: fastestimator/cli/train.py [hunk 1]
```diff
@@ -88,12 +91,13 @@ def configure_train_parser(subparsers: argparse._SubParsersAction) -> None:
                         dest='hyperparameters_json',
                         type=str,
                         help="The path to the hyperparameters JSON file")
-    parser.add_argument('--warmup', type=str, help="Warmup setting, can be true, false or debug", default="true")
+    parser.add_argument('--warmup', type=str, help="Warmup setting, can be true or false", default="true")
```
Comment (TortoiseHam) [inline fastestimator/cli/train.py:94]: we could switch this to boolean type rather than going thru string parsing

Comment (vbvg2008) [reply fastestimator/cli/train.py:94]: smart

File: test/PR_test/unit_test/dataset/test_data.py [hunk 2]
```diff
@@ -41,7 +41,9 @@ def test_dataset_urls(self):
             if isinstance(value, list):
                 for url in value:
                     with self.subTest('{}{} url'.format(key, url)):
-                        self.assertTrue(ping(url))
+                        if not ping(url):
+                            print("Warning!!: {} is not reachable at {}".format(key, url))
```
Comment (TortoiseHam) [inline test/PR_test/unit_test/dataset/test_data.py:45]: Is there something with more oomf than a print message that we can do here?

Comment (vbvg2008) [reply test/PR_test/unit_test/dataset/test_data.py:45]: like print("WARNING!!!!!!!!!!!!!!!") ?

Comment (TortoiseHam) [reply test/PR_test/unit_test/dataset/test_data.py:45]: lol. like something that would show up in the test summary as neither passed nor failed.

Comment (vbvg2008) [reply test/PR_test/unit_test/dataset/test_data.py]: The warning is in colored font now, pretty noticeable.  Didn't find anything that can show up in the end tho.

**Review by TortoiseHam — APPROVED** [review]
LG

**Pull request merged**